packages/react/src/experimental/SelectPanel2/SelectPanel.tsx

```diff
@@ -309,8 +312,13 @@ const Panel: React.FC<SelectPanelProps> = ({
                 '--max-height': maxHeightValue,
                 '--position-top': `${position?.top ?? 0}px`,
                 '--position-left': `${position?.left ?? 0}px`,
+                visibility: internalOpen ? 'visible' : 'hidden',
+                display: 'flex',
               } as React.CSSProperties)
-            : undefined
+            : {
+                visibility: internalOpen ? 'visible' : 'hidden',
+                display: 'flex',
+              }
```

Comment:
and these I think could go inside the sx?

Reply:
Unfortunately they have to be set as the `style` attribute to override the browser's default stylesheet 😢